### ISSUE TITLE
Clickly expanding leaves children blank

### ISSUE THREAD

**Issue by mshd** [trongthanh/family-tree#2]
In the [demo](https://int3ractive.com/family-tree/) collapsing quickly leaves the children nodes blank

**Comment by trongthanh** [comment]
Good catch. However, this side project is current not my priority so I'll try to fix as soon as I can save some time for it.

**Comment by bmrejen** [comment]
Click slower and the problem won't happen
